Tasks/cURLUploader/curluploader.ts

```diff
@@ -86,12 +86,10 @@ async function run() {
             // Now we get a list of all files under this root
             var allFiles = tl.find(findPathRoot);
 
-            // IMPORTANT: The backslash character is a valid character in linux style directory and file names. However, as
-            // this is a file pattern not a file path we are converting all backslashes to forward slashes.
-            filesPattern = filesPattern.replace(/\\/g, '/');
-
             // Now matching the pattern against all files
-            var uploadFilesList = tl.match(allFiles, filesPattern, undefined, {matchBase: true});
+            var uploadFilesList = tl.match(allFiles, filesPattern, undefined, {matchBase: true}).map( (s) => {
```

Comment:
Please add a comment on why we had to do this.

Reply:
done. Thanks!